api/ws/rpc/connect.js

```diff
@@ -45,20 +45,19 @@ const connectSteps = {
 	addConnectionOptions: peer => {
 		const systemHeaders = System.getHeaders();
 		const queryParams = {};
-
-		if (systemHeaders.version != null) {
```

Comment:
The logic is changed here, previously for every undefined and null a header wasn't assigned, now it will not be assigned for every falsy value ("", NaN, 0, false in addition).